### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by weimengting** [liuf1990/3DFC#10]
Do you have plan to release the training code? It is crucial to understand the loss functions defined in the paper.
